docs/src/main/paradox/api-reference/schemas.md

```diff
@@ -1,252 +1,153 @@
-| domainId      | String        | The unique identifier for the domain for this particular organization.                                                        |
-| name          | String        | The unique identifier for the schema for this particular domain.                                                              |
-| version       | String        | The unique identifier for the schema version for this particular schema, following the [semantic format](http://semver.org/). |
-| fragment      | String        | The unique short identifier for a shape inside the schema.                                                                    |
-
-
-
-### Response JSON-LD Object
-| Field         | Type          | Description                                   |
-| ------------- |-------------  | --------------------------------------------- |
-| @id           | String        | The unique identifier for the shape.          |
-| rev           | Long          | The current revision of the schema.           |
-| deprecated    | Boolean       | whether the schema is deprecated or not.      |
-| published     | Boolean       | whether the schema is deprecated or not.      |
-| *             | *             | The payload of the shape.                     |
-
-### Status Codes
-* **200 OK** - Request completed successfully.
-* **404 Not Found** - The requested shape does not exist.
-
-### Example request
-```bash
-curl -v "https://bbp-nexus.epfl.ch/{environment}/{version}/schemas/nexus/core/subject/v1.0.0/shapes/AgeValueShape"
+#### Example
+
+Request
+:   @@snip [schema-delete.sh](../assets/api-reference/schemas/schema-delete.sh)
+
+Response
+:   @@snip [schema-ref-new.json](../assets/api-reference/schemas/schema-ref-delete.json)
+
+### Search schemas
+
+Follows the general definition of searching in a collection of resources.
+
 ```
+GET /v0/schemas/{orgId}/{domId}/{name}
+      ?q={full_text_search_query}
+      &filter={filter}
+      &from={from}
+      &size={size}
+      &deprecated={deprecated}
+      &published={published}
+```
+... where 
+
+* `{orgId}` filters the resulting schemas to belong to a specific organization.
+* `{domId}` filters the resulting schemas to belong to a specific domain.
+* `{name}` filters the resulting schemas to have a specific name.
+* `{full_text_search_query}` is an arbitrary string that is looked up in the attribute values of the selected schemas.
+* `{filter}` is a filtering expression as described in the @ref:[Search and filtering](operating-on-resources.md#search-and-filtering) section.  
+* `{from}` and `{size}` are the listing pagination parameters.  
+* `{deprecated}` selects only schemas that have the specified deprecation status.
+* `{published}` selects only schemas that have the specified published status.
+
+All query parameters described (`q`, `filter`, `from`, `size`, `deprecated` and `published`) are optional.
+
+The path parameters `/{orgId}/`, `/{domId}/` and `/{name}/` are optional.
```

Comment:
The path segments ({orgId}/{domId}/{name}) are optional; when used, they constrain the resulting listing to contain only schemas that share the same organization, domain ids and name. Any of the segments can be omitted but since they are positional it's required that depended segments (to the left) are specified. For example, one can list all the schemas within a domain using a simple GET request on /v0/schemas/{orgId}/{domId}.